includes/admin/class-wc-admin-notices.php

```diff
@@ -341,6 +343,36 @@ public static function regenerating_lookup_table_notice() {
 		include dirname( __FILE__ ) . '/views/html-notice-regenerating-lookup-table.php';
 	}
 
+
+	/**
+	 * If on WordPress 5.0 or greater, inform users of WooCommerce Admin feature plugin.
+	 *
+	 * @since 3.6.4
+	 * @todo  Remove this notice and associated code once the feature plugin has been merged into core.
+	 */
+	public static function add_wc_admin_feature_plugin_notice() {
+		$wordpress_version            = get_bloginfo( 'version' );
+
+		if ( version_compare( $wordpress_version, '4.9.9', '>' ) ) {
```

Comment:
Although it's unlikely, it is possible for there to be a WP 4.9.10 due to security releases. If this were changed to 

`version_compare( $wordpress_version, '5.0', '>=' )`

then that situation would be avoided.

Reply:
Good call Ron, thank you.